.github/workflows/build.yaml

```diff
@@ -29,4 +29,18 @@ jobs:
 
       - name: Build with Maven
         run: mvn package
+      
+      # Extract version from pom.xml
+      - name: Get Version from POM
```

Comment:
The comments don't really add anything that isn't explained by the step name. I'd probably remove them.